setup.py

```diff
@@ -40,7 +40,9 @@
     'pytest-runner>=2.6.2',
 ]
 
-install_requires = []
+install_requires = [
+    'requests==2.18.4',
```

Comment:
Use as in https://github.com/inveniosoftware/invenio-opendefinition/blob/e6379f96307bc2a75cc8771862d4fc278f962dfa/setup.py#L88